.chloggen/reencrypt_gw_default.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Is it maybe breaking? Since some certs will change.

Reply:
I don't think this is a breaking change, as the other things will remaing working in the same way. I think this only adds a couple of hosts to the cert.